packages/components/src/modifiers/hds-clipboard.ts

```diff
@@ -118,17 +119,23 @@ export const writeTextToClipboard = async (
       // console.log('success', textToCopy);
       return true;
     } catch (error) {
-      // clipboard write failed
-      // this probably never happens (see comment above) or happens only for very old browsers that don't for which `navigator.clipboard` is undefined
-      warn(
-        `copy action failed, please check your browser‘s permissions: ${JSON.stringify(
-          error
-        )}`,
-        {
-          id: 'hds-clipboard.write-text-to-clipboard.catch-error',
-        }
-      );
-      return false;
+      // if it is not a secure context, use the polyfill
+      // to test that this works in a non-secure context, access the port through your IP address (ie. XXX.XXX.X.XXX:4200/)
+      if (!navigator.clipboard) {
+        await clipboard.writeText(textToCopy);
```

Comment:
[question] for my education, does this mean we are effectively lazy loading the polyfill or is it loaded 100% of the time regardless?

Reply:
I think its loaded 100% of the time, but only used if its not a secure context.

Reply:
I'd be happy to 🍐  to explore this a bit, but maybe @alex-ju would know if lazy loading it would be possible in this instance?

Reply:
I've not looked at the code yet, but in theory yes, we can (and probably should) lazy load via `ember-auto-import`. I'll try and review this later today